src/Illuminate/Console/Command.php

```diff
@@ -321,36 +332,71 @@ public function table(array $headers, $rows, $style = 'default')
     }
 
     /**
+     * Check if verbosity level is in bounds.
+     *
+     * @param  string|int  $level
+     * @return bool
+     */
+    public function inVerbosity($level)
+    {
+        if (! is_int($level) && isset($this->verbosity[$level])) {
+            $level = $this->verbosity[$level];
+        } else {
+            $level = OutputInterface::VERBOSITY_NORMAL;
```

Comment:
Ah, you're right. Was about to re-write this method and forgot to add the ! is_int back.